x-pack/plugins/lists/server/services/exception_lists/utils.ts

```diff
@@ -80,7 +82,7 @@ export const transformSavedObjectToExceptionList = ({
     namespace_type: namespaceType,
     tags,
     tie_breaker_id,
-    type,
+    type: exceptionListType.is(type) ? type : 'detection',
```

Comment:
This is a result of both the list and item `type` being mapped to the same `type` property in the SO. We can discuss whether to keep this as is (@FrankHassanabad are there downsides to separating these out?) or change.